main.py

```diff
@@ -272,19 +294,17 @@ def command_list_devices(self, update: Update, context: CallbackContext) -> None
         self.list_devices(update, context, devices, None)
 
     def command_list_groups(self, update: Update, context: CallbackContext) -> None:
-        if not self.is_admin(update, context=context, log_warning=True):
-            return
-        group_filter = ' '.join(context.args).lower()
+        self.request_access(update, context, AccessLevel.ADMIN)
```

Comment:
This looks like a good case for python decorator function.  However, I am not sure if this is really going to be a win for this code - decorator is one line, this invocation is also one line of code, but using decorator for access is more pythonic.  I guess you can entertain yourself if you like.

Reply:
Good one. Created https://github.com/vdbg/hubibot/issues/19 to make the change in a separate pr